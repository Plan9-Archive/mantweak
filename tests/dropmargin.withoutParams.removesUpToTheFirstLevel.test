#!/bin/rc
expectedOUT=$0.out
result=`{cmp <{dropmargin $0.in} <{cat $expectedOUT}}
rsize=`{echo $result|wc -r}
if(~ $rsize 0){}
if not {
	echo $0 failed: $"result;
	exit "fail";
}
