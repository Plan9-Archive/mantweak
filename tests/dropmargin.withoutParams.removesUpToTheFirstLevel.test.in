
	Here end the margin (1 tab)

	  Here are 2 space more for a second layer
        Here the margin is replaced by 8 spaces


	     Here are 5 space more for a third layer