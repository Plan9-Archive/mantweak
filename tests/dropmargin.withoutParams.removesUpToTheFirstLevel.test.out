
Here end the margin (1 tab)

  Here are 2 space more
Here the margin is replaced by 8 spaces (assuming $tabstop=8)
